### PR TITLE
RickrollRunnable + Website updates

### PR DESCRIPTION
**Pull request by Tiger-The-Cat**
# Your checklist for this pull request

## Author name

**Author:** Tiger-The-Cat

## About your game

**What is your game about?**
This is an updated version of rickroll.js, it has been shortened to fit on the raspberry pi pico.

**How do you play your game?**
Running the game will start the animation.

## Code
Check off the items that are true. 
- [x] The game was made using the [Sprig editor](https://sprig.hackclub.com/editor).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. 

> Thanks for your PR!

### CODE REVIEW

**Review by LucasHT22 — APPROVED** [review]
Great! Perfect run on the console:)
I'll ask someone else to review your changes on the site (I can only merge games)

File: src/pages/gallery/index.astro [hunk 1]
```diff
@@ -202,7 +204,7 @@ games
 			<div class='info-split'>
 				<div class='search-container'>
 					<p>
-						We currently have {games.length} games made my Hack Clubbers. Click on any to play in the browser and hack on the code in our editor!
+						We currently have {games.length} games made by Hack Clubbers. Click on any to play in the browser and hack on the code in our editor!
```
Comment (kognise) [inline src/pages/gallery/index.astro:207]: Great catch! Thanks!

File: src/pages/index.astro [hunk 2]
```diff
@@ -82,14 +88,14 @@ if (session && session.session.full) {
...
 							</div>
 						</article>
 						<article>
 							<img src='/icons/display.png' alt='Display icon' />
 							<div>
-								<h3>Display &ndash TFT7735</h3>
+								<h3>Display - TFT7735</h3>
 								<p>160&times128 color LCD</p>
 							</div>
 						</article>
```
Comment (kognise) [inline src/pages/index.astro:101]: I think it's better to have the HTML entity, more clear in IDEs. (Same in line 112 but GitHub wouldn't let me highlight that too.)

**Review by kognise — CHANGES_REQUESTED** [review]
Thanks for your site contributions! In the future it'd be better to separate your PRs, with game in one and website updates in another, but I think it's alright to merge these together. Left a couple notes :)

File: src/pages/index.astro (hunk 2 above)
Comment (Tiger-The-Cat) [reply src/pages/index.astro:101]: I agree, the only problem is that those elements currently look like so:
image

File: src/pages/gallery/index.astro [hunk 3]
```diff
@@ -192,6 +192,8 @@ games
 				font-size: 0.9rem;
 			}
 		</style>
+		
+    <style>
```
[the remark this answers is not in the document]
Comment (Tiger-The-Cat) [reply src/pages/gallery/index.astro:196]: Yes, sorry. That is a typo.

File: src/pages/gallery/index.astro (hunk 1 above)
Comment (Tiger-The-Cat) [reply src/pages/gallery/index.astro:207]: No problem!

File: src/pages/index.astro [hunk 4]
```diff
@@ -34,6 +34,12 @@ if (session && session.session.full) {
 	<head>
 		<StandardHead title={null} />
 		<title>Sprig</title>
+		<link rel="preload" as="image" href='https://sprig.hackclub.com/stories-big/sprig-front.jpeg' />
+		<link rel="preload" as="image" href='https://sprig.hackclub.com/stories-big/sprig-back.jpeg' />
+		<link rel="preload" as="image" href='https://sprig.hackclub.com/stories-big/play.jpeg' />
+		<link rel="preload" as="image" href='https://sprig.hackclub.com/stories-big/develop.jpeg' />
+		<link rel="preload" as="image" href='https://sprig.hackclub.com/stories-big/orpheus.jpeg' />
+		<link rel="preload" as="image" href='https://sprig.hackclub.com/stories-big/learn.jpeg' />
```
[the remark this answers is not in the document]
Comment (Tiger-The-Cat) [reply src/pages/index.astro:42]: Sure!

**Comment by grymmy** [comment]
Closing this PR due to lack of activity.  Please reopen it if you'd like to continue forward.

**Pull request closed without merging**